types/azdata/index.d.ts

```diff
@@ -2116,7 +2118,7 @@ declare module 'azdata' {
          * Launches a flyout dialog that will display the information on how to complete device
          * code OAuth login to the user. Only one flyout can be opened at once and each must be closed
          * by calling {@link endAutoOAuthDeviceCode}.
-         * @param providerId    ID of the provider that's requesting the flyout be opened
+         * @param providerId	ID of the provider that's requesting the flyout be opened
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/azdata/index.d.ts;
  Line: 2121;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.